server/render/index.js

```diff
@@ -74,8 +75,40 @@ export function render( element, key = JSON.stringify( element ) ) {
 	//todo: render an error?
 }
 
+function getCacheKey( context ) {
```

Comment:
Should this helper live inside `server/isomorphic-routing` since it's specific to that case?